config/routes.rb

```diff
@@ -1,6 +1,15 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  resources :comments
-  resources :dependencies
+  concern :resources do
+    resources :dependencies
+    resources :comments
+  end
+
+  scope format: ENV.key?("ALLOW_FORMAT_FROM_URL2") do
```

Comment:
`ALLOW_FORMAT_FROM_URL2` => `ALLOW_FORMAT_FROM_URL` (remove the extra `2`)

Reply:
Good catch, it's now fixed :heavy_check_mark: